schemata.services.yml

```diff
@@ -1,22 +1,25 @@
 services:
+  serializer.encoder.json_schema:
+    class: Drupal\schemata\Encoder\JsonSchemaEncoder
+    tags:
+      - { name: encoder, priority: 22, format: 'schema_json' }
```

Comment:
I was just curious if this was new, do we know why it's 22 vs. 20. We can figure it out later.